test/components/views/context_menus/MessageContextMenu-test.tsx

```diff
@@ -13,35 +13,220 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Could you localize this to the spot in the file that needs it, rather than disabling the rule for the whole file?